signalr-node-server/signalr.js

```diff
@@ -133,9 +133,10 @@ class HubConnection {
 
             if (!protocol) {
                 // Fail for anything but JSON right now
-                this._doHandshakeResponse(`Requested protocol '${handshakeMessage.protocol}' is not available.`);
-            }
-            else {
+                this._doHandshakeResponse(
+                    `Requested protocol '${handshakeMessage.protocol}' is not available.`
```

Comment:
Why this formatting change?

Reply:
Ah this is caused by the `.prettierrc` file `printWidth` setting I guess (max 100 chars per line)